bookied/cli.py

```diff
@@ -80,12 +80,12 @@ def worker(queue):
         if q.count > 0:
             log.info("Emptying Redis Queue (default) ...")
             q.empty()
-            job = q.dequeue()
+            job = q.delete()
             while job is not None:
                 log.info("Canceling " + str(job))
                 job.cancel()
                 job.delete()
-                job = q.dequeue()
+                job = q.delete()
```

Comment:
Here the purpose is to cancel and delete one by one from the list. Could you delete everything without cancel?

Reply:
q doesn't have a method dequeue associated with it and things worked well with delete.